libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/LanguageGeneratorManager.cs

```diff
@@ -53,6 +53,12 @@ public LanguageGeneratorManager(ResourceExplorer resourceExplorer)
         public ConcurrentDictionary<string, LanguageGenerator> LanguageGenerators { get; set; } = new ConcurrentDictionary<string, LanguageGenerator>(StringComparer.OrdinalIgnoreCase);
 #pragma warning restore CA2227 // Collection properties should be read only
 
+        /// <summary>
+        /// Returns the resolver to resolve LG import id to template text based on language and a template resource loader delegate.
+        /// </summary>
+        /// <param name="locale">Locale to identify language.</param>
+        /// <param name="resourceMapping">Template resource loader delegate.</param>
+        /// <returns>resolver.</returns>
```

Comment:
The return could be changed to something better
```suggestion
        /// <returns>An instance of <see cref="ImportResolverDelegate">.</returns>
```

O
```suggestion
        /// <returns>The delegate to resolve the resource.</returns>
```